Formula/soccerwindow2.rb

```diff
@@ -16,12 +16,17 @@ class Soccerwindow2 < Formula
   depends_on "librcsc"
 
   def install
+    # ld: framework not found Security-lz
+    inreplace "configure",
```

Comment:
Heh, this is identical to the problem is another of the upstream packages that [I patched](https://github.com/Homebrew/homebrew-core/blob/master/Formula/rcssserver.rb#L70-L77) a while back now.